cmd/app/options/options.go

```diff
@@ -200,6 +200,8 @@ func (o *Options) addTLSFlags(fs *pflag.FlagSet) {
 		"serving-certificate-key-size", 2048,
 		"Number of bits to use for the server's serving certificate (RSAKeySize).")
 
+	fs.BoolVar(&o.TLS.ServingUseECAlgorithm, "serving-use-ec-algorithm", false,
+		"Use Elliptical Signature algorithm to use when generating private keys.")
```

Comment:
We should change this to be a string with accepts either `"RSA"` or `"ECDSA"` which defaults to `"RSA"`.

Reply:
Done. Btw: do you think it's worth to add validation to one of two?

Reply:
Yes please!